File: src/main/java/spg/client/model/Current.java
package spg.client.model;

import javafx.beans.property.SimpleIntegerProperty;
import javafx.beans.property.SimpleStringProperty;

public class Current {
    public static final SimpleStringProperty panel = new SimpleStringProperty(
        Settings.mainTitle.get()
    );

    public static final SimpleIntegerProperty server = new SimpleIntegerProperty(
        -1
    );

    public static final SimpleIntegerProperty channel = new SimpleIntegerProperty(
        -1
    );
}

File: src/main/java/spg/client/model/Current.kt
package spg.client.model

import javafx.beans.property.SimpleIntegerProperty
import javafx.beans.property.SimpleObjectProperty
import javafx.beans.property.SimpleStringProperty
import spg.client.model.Settings.ServerListItem

object Current {
	val panel = SimpleStringProperty(
		Internal.mainTitle.get()
	)

	val server = SimpleObjectProperty<ServerListItem?>(
		null
	)

	val channel = SimpleIntegerProperty(
		-1
	)
}